config/gatekeeper/admissionregistration.k8s.io_v1_mutatingwebhookconfiguration_gatekeeper-mutating-webhook-configuration.yaml

```diff
@@ -1,6 +1,7 @@
 apiVersion: admissionregistration.k8s.io/v1
 kind: MutatingWebhookConfiguration
 metadata:
+  creationTimestamp: null
```

Comment:
why createtimestamp added with null?

Reply:
This was added in upstream Gatekeeper so when I ran `make import-manifests` it got added:
https://github.com/open-policy-agent/gatekeeper/blob/1f6f701644ba11046074fe6675127151e91b7773/config/webhook/manifests.yaml#L5

Other YAML files already had this set, so I assumed it was benign to have it rather than have a Kustomize patch to remove it.